Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -24,66 +47,177 @@ public partial class GestureManager : Singleton<GestureManager>
         /// set the override focused object.
         /// If its null, then the gazed at object will be selected.
         /// </summary>
-        public GameObject OverrideFocusedObject
-        {
-            get; set;
-        }
-
+        public GameObject OverrideFocusedObject { get; set; }
+        
         /// <summary>
         /// Gets the currently focused object, or null if none.
         /// </summary>
-        public GameObject FocusedObject
+        public GameObject FocusedObject { get; private set; }
+
+        /// <summary>
+        /// Whether or not a manipulation gesture is currently in progress
+        /// </summary>
+        public bool ManipulationInProgress { get; private set; }
+
+        /// <summary>
+        /// The offset of the hand from its position at the beginning of 
+        /// the currently active manipulation gesture, in world space.  Not valid if
+        /// a manipulation gesture is not in progress
+        /// </summary>
+        public Vector3 ManipulationOffset { get; private set; }
+
+        /// <summary>
+        /// The world space position of the hand being used for the current manipulation gesture.  Not valid
+        /// if a manipulation gesture is not in progress.
+        /// </summary>
+        public Vector3 ManipulationHandPosition
         {
-            get { return focusedObject; }
+            get
+            {
+                Vector3 handPosition = Vector3.zero;
+                currentHandState.properties.location.TryGetPosition(out handPosition);
+                return handPosition;
+            }
         }
 
         private GestureRecognizer gestureRecognizer;
-        private GameObject focusedObject;
+        private GestureRecognizer manipulationRecognizer;
+
+        private bool HandPressed { get { return pressedHands.Count > 0; } }
+        private HashSet<uint> pressedHands = new HashSet<uint>();
+
+        private InteractionSourceState currentHandState;
 
         void Start()
         {
+            InteractionManager.SourcePressed += InteractionManager_SourcePressed;
```

Comment:
Quite possible that I'm missing something, but it looks like the GestureManipulator is only using the hand position to compute the delta from the initial hand position. But isn't that delta just the value of the ManipulationOffset converted to the camera space? If so we can drop tracking of the hand position entirely and still get the functionality of the GestureManipulator.

Does that make sense?